File: src/main/java/com/example/ssauc/user/bid/repository/ProductReportRepository.java
package com.example.ssauc.user.bid.repository;

import com.example.ssauc.user.bid.entity.ProductReport;
import com.example.ssauc.user.login.entity.Users;
import org.springframework.data.domain.Page;
import org.springframework.data.domain.Pageable;
import org.springframework.data.jpa.repository.JpaRepository;

public interface ProductReportRepository extends JpaRepository<ProductReport, Long> {
    Page<ProductReport> findByReporter(Users reporter, Pageable pageable);
}

File: src/main/java/com/example/ssauc/user/bid/repository/ProductReportRepository.kt
package com.example.ssauc.user.bid.repository

import com.example.ssauc.user.bid.entity.ProductReport
import com.example.ssauc.user.login.entity.Users
import org.springframework.data.domain.Page
import org.springframework.data.domain.Pageable
import org.springframework.data.jpa.repository.JpaRepository

interface ProductReportRepository : JpaRepository<ProductReport?, Long?> {
    fun findByReporter(reporter: Users?, pageable: Pageable?): Page<ProductReport?>?
}